src/bin/gui/main_gui.ml

```diff
@@ -979,22 +969,18 @@ let search_all entry (_sv:GSourceView2.source_view)
       search_one buf str result iter found_all_tag
     done
 
-
 let start_gui all_used_context =
   Options.set_timers true;
   Options.set_thread_yield Thread.yield;
 
   (* TODO: crash : change this*)
   set_timeout (fun () ->
-      Printer.print_std "Timeout";
+      Printer.print_std "Timeout TODO";
```

Comment:
`TODO` should either be handled now or appear as a comment, but not as a user feedback